push-rewrite

```diff
@@ -71,7 +66,7 @@ def main():
     if not tests_disabled:
         label(pull, {"labels": labels + ["no-test"]})
 
-    git("push", "--force-with-lease=" + opts.branch + ":" + remote, opts.remote, opts.branch + ":" + opts.branch)
+    git("push", "--force-with-lease")
```

Comment:
There's a small opportunity that this will push to origin, if people configured "origin" to be writable and the actual cockpit-project/cockpit. That's unexpected (but not technically wrong). All in all I think that's fine.